op-chain-ops/upgrades/l1.go

```diff
@@ -305,18 +305,35 @@ func L2OutputOracle(batch *safe.Batch, implementations superchain.Implementation
 		return err
 	}
 
```

Comment:
This logic is more complex that I would prefer. You are following the way that is was done previously which is good. 

What do you think about instead of pulling from the config when its present, always pull from the remote and then if the config is present, throw an error if there is a mismatch. The reasoning here would be that we are not doing config changes are part of this upgrade, we want to reuse the values from the chain and only use the config for new values. The error throwing would make us update the configs so that they match what is running in prod. Going into the future, we want to always use the config when doing upgrades in `superchain-ops`